extension/src/extension.test.ts

```diff
@@ -277,12 +291,16 @@ describe("S3 VS Extension", () => {
   describe("deleteInstance()", () => {
     it("drops connection and virtual schema", () => {
       const context = createMockContext();
-      createExtension().deleteInstance(context, "instId")
+      createExtension().deleteInstance(context, CONFIG.version, "instId")
       const executeCalls = context.executeMock.mock.calls
       expect(executeCalls.length).toEqual(2)
       expect(executeCalls[0]).toEqual(["DROP VIRTUAL SCHEMA IF EXISTS \"instId\" CASCADE"])
       expect(executeCalls[1]).toEqual(["DROP CONNECTION IF EXISTS \"instId_CONNECTION\""])
     })
+    it("fails for wrong version", () => {
```

Comment:
How can user get rid of old instances, if a new version cannot remove instances of old version? 
Is there a uninstall procedure?

Reply:
Good question. Once we publish another version, we need to implement and test the uninstaller logic for all previous versions. For now that's not yet necessary.